corehq/apps/domain/views.py

```diff
@@ -1793,7 +1793,7 @@ def post(self, request, *args, **kwargs):
 
                     m_file.save()
 
-            ignore = []
+            ignore = ['UserRole']
```

Comment:
in other news, I'd like to nominate this particular method as a candidate for the "refactor this code" interview question.  @benrudolph thoughts?